app/helpers/application_helper.rb

```diff
@@ -19,6 +19,14 @@ def controller_t(lazy_t, *args)
     t ".#{[*controller, lazy_t].join('.')}", *args
   end
 
+  def menu_button
+    return if user_header_link.nil?
```

Comment:
I think the decision to show the menu button belongs in the view. So I'd suggest removing this link and in the view doing:
```erb
<%= menu_button if user_header_link.nil? %>
```

Reply:
Thanks 👍 

I did <%= menu_button unless user_header_link.nil? %> because it only shows for providers

Reply:
would `<%= menu_button unless user_header_link %>` work?

Reply:
Personally - I prefer `unless user_header_link` but some devs coming from other code languages struggle with that bit of Ruby sugar and prefer the more explicit `.nil?` check. So I'm happy with either.

Reply:
Abdi, I see you just merged this PR with `<%= menu_button unless user_header_link.nil? %>`
Which is the opposite of the suggestion from Rob and I
If `unless user_header_link.nil?` is the right condition, then `if user_header_link` would be simpler